com.locomalito.abbayedesmorts.appdata.xml

```diff
@@ -0,0 +1,72 @@
+<?xml version="1.0" encoding="utf-8"?>
+<component type="desktop">
+  <id>com.locomalito.abbayedesmorts</id>
+  <metadata_license>CC-BY-3.0</metadata_license>
+  <project_license>GPLv3</project_license>
+  <content_rating type="oars-1.1" />
+  <name>L'Abbaye des morts</name>
+  <summary>An obsolete video game for a dark passage of history</summary>
+  <description>
+      <p>
+          In the 13th century, the Cathars, clerics who preached about the poverty of Christ and defended life without material aspirations, were treated as heretics by the Catholic Church and expelled out of the Languedoc region in France. One of them, called Jean Raymond, found an old church in which to hide from crusaders, not knowing that beneath its ruins lay buried an ancient evil.
+      </p>
+      <p>
+          Faith will be your only weapon in this platformer styled like a ZX Spectrum game. Black backgrounds, 1 color sprites and 1 bit sounds are a proper fit for a raw story. The lack of details turn on the player's imagination, creating a unique experience for each player.
+      </p>
+      <ul> Features
+      <li>Look and feel of a ZX Spectrum game.</li>
+      <li>23 screens to explore.</li>
+      <li>Riddles and hints to find items.</li>
+      <li>Around 30 minutes of game length (once fully mastered).</li>
+  </ul>
+  </description>
+  <developer_name>LocoMalito</developer_name>
+  <url type="homepage">https://www.locomalito.com</url>
```

Comment:
I would add a bugtracker url here.

Reply:
Shouldn't that point to the project's homepage? I could add the github one here instead
https://github.com/nevat/abbayedesmorts-gpl